source/core/TDCodecVia.cpp

```diff
@@ -2329,6 +2329,50 @@ void TDViaFormatter::FormatClusterData(TypeRef type, void *pData)
     _string->Append(Fmt()._clusterPost);
 }
 //------------------------------------------------------------
+void TDViaFormatter::FormatVariant(TypeRef type, void *pData)
+{
+    _string->Append(Fmt()._clusterPre);
```

Comment:
Variants are represented as follows:
`{"value": <expand-underlying-type>, "attributes":["foo": <expand-attribute-value>, "bar": <expand attribute value>]}`

When either of these fields are null, you will see something like this:
`{"value": null, "attributes": null}`